chapters/statements.tex

```diff
@@ -34,11 +34,11 @@ \subsection{An Algorithm in a Model}\label{execution-of-an-algorithm-in-a-model}
 \item
   A continuous-time variable is initialized with the value of its \lstinline!start!-attribute.
 \item
-  A discrete-time variable \lstinline!v! is initialized with \lstinline!pre(v)!.
+  A discrete-time variable \lstinline!v! in a non-initial algorithm is initialized with \lstinline!pre(v)!.
 \item
   If at least one element of an array appears on the left hand side of the assignment operator, then the complete array is initialized in this algorithm section.
 \item
-  A parameter assigned in an initial algorithm, \cref{initialization-initial-equation-and-initial-algorithm}, is initialized with the value of its \lstinline!start!-attribute.
+  In an initial algorithm, \cref{initialization-initial-equation-and-initial-algorithm}, any variable (including a parameter) is initialized with the value of its \lstinline!start!-attribute.
```

Comment:
Similarly:
```suggestion
  Within the initialization problem treatment of an algorithm, \cref{initialization-initial-equation-and-initial-algorithm}, any variable (including a parameter) is initialized with the value of its \lstinline!start!-attribute.
```

Reply:
No, as indicated that change would be incorrect and break things.